addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/OpenApiAPI.java

```diff
@@ -143,6 +131,19 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
         }
     }
 
+    private static ApiResponseList createResponse(String name, List<String> errors)
+            throws ApiException {
+        if (!errors.isEmpty()) {
+            throw new ApiException(ApiException.Type.BAD_EXTERNAL_DATA, String.join(",", errors));
+        }
+
+        ApiResponseList result = new ApiResponseList(name);
+        if (errors.isEmpty()) {
```

Comment:
this isnt needed, is it? We're checking for `! errors.isEmpty()` above..

Reply:
Yes you're right it could be an else, or probably just fall through.